### PR TITLE
Rename the library to "mint"

### PR DESCRIPTION
**Pull request by whatyouhide**

### CODE REVIEW

File: test/mint/http1/conn_test.exs [hunk 1]
```diff
@@ -1,63 +1,62 @@
-defmodule XHTTP1.XHTTP1Test do
+defmodule Mint.HTTP1.Mint.HTTP1Test do
```
Comment (ericmj) [inline test/mint/http1/conn_test.exs]:
```suggestion
defmodule Mint.HTTP1Test do
```

File: test/mint/http2/conn_test.exs [hunk 2]
```diff
@@ -1,9 +1,9 @@
-defmodule XHTTP2.XHTTP2Test do
+defmodule Mint.HTTP2.Mint.HTTP2Test do
```
Comment (ericmj) [inline test/mint/http2/conn_test.exs]:
```suggestion
defmodule Mint.HTTP2Test do
```

**Comment by ryanwinchester** [comment]
@whatyouhide where does Mint come from? 🤣

**Comment by whatyouhide** [comment]
@ryanwinchester it just came out of a few names we were throwing around. "Mint" like fresh or something like that 😄

**Pull request merged**